Content.Shared/Storage/Components/SecretStashComponent.cs

```diff
@@ -7,54 +7,89 @@
 using Robust.Shared.GameStates;
 using Content.Shared.DoAfter;
 using Robust.Shared.Serialization;
+using Robust.Shared.Audio;
 
 namespace Content.Shared.Storage.Components
 {
     /// <summary>
     ///     Logic for a secret slot stash, like plant pot or toilet cistern.
-    ///     Unlike <see cref="ItemSlotsComponent"/> it doesn't have interaction logic or verbs.
-    ///     Other classes like <see cref="ToiletComponent"/> should implement it.
+    ///     Unlike <see cref="ItemSlotsComponent"/> it has logic for opening and closing
+    ///     the stash.
     /// </summary>
     [RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
     [Access(typeof(SecretStashSystem))]
     public sealed partial class SecretStashComponent : Component
     {
         /// <summary>
-        ///     Max item size that can be fitted into secret stash.
+        ///     Max item size that can be inserted into secret stash.
```

Comment:
this can be removed entirely now i think item slots directly supports max item size

Reply:
Yeah it technically isn't needed right now but I want redo #25306 and It's necessary for that. I don't think combining that with this PR make sense. Should I just remove this for now and re-add it in when I remake the other PR?

Reply:
I couldn't figure this out! Could you give me an example of it being used? I looked at the container system and it didn't seem to have any functions that work like that.

Reply:
I don't think this is possible? I looked and I don't think item slots do support max item size! I'm just going to resolve this now but if can show me how I'll do it!

Reply:
I couldn't spot anything so